packages/breakdown-trees/package.json

```diff
@@ -0,0 +1,141 @@
+{
+  "name": "@bentley/breakdown-trees-react",
```

Comment:
@calebmshafer should this be itwin scoped since this repo is going to be deprecated soon?

Reply:
yes, but when it does I think we should bundle that with it consuming 3.0 and the new `@itwin` based packages. Otherwise we'll just introduce another breaking change.